### PR TITLE
✨ Source Zendesk Support: add new stream user_fields

### PR DESCRIPTION
**Pull request by dheerendravarma**
## What
Added the support for new stream `user_fields` in Zendesk Support. https://github.com/airbytehq/airbyte/issues/19123

## How
- Added the new schema file for user_fields stream in `/schemas/user_fields.json`

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*
No

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests#pull-request-title-convention)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/user_fields.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "properties": {
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/user_fields.json:2]: Please update the schema following the reference here: https://docs.airbyte.com/connector-development/schema-reference

File: airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json [hunk 2]
```diff
@@ -27759,7 +27759,7 @@
     "sourceDefinitionId": "79c1aa37-dae3-42ae-b333-d1c105477715",
     "name": "Zendesk Support",
     "dockerRepository": "airbyte/source-zendesk-support",
-    "dockerImageTag": "0.2.29",
+    "dockerImageTag": "0.2.30",
```
Comment (marcosmarxm) [inline airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json:27762]: A new stream is a minor update following the docs here: https://docs.airbyte.com/contributing-to-airbyte/#examples please fix it.

File: airbyte-integrations/connectors/source-zendesk-support/Dockerfile [hunk 3]
```diff
@@ -25,5 +25,5 @@ COPY source_zendesk_support ./source_zendesk_support
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.29
+LABEL io.airbyte.version=0.2.30
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-support/Dockerfile:28]: Please udpate the metadata.yaml file inside the connector folder too.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by marcosmarxm** [comment]
@dheerendravarma are you planning to continue this contribution?

**Comment by dheerendravarma** [comment]
@marcosmarxm I will continue this contribution. I will make the changes as per your feedback.

**Comment by marcosmarxm** [comment]
Closed due inactivity. @dheerendravarma please let me know if you want to return this work in the future.

**Pull request closed without merging**